### PR TITLE
Updated Prima db schema

### PR DESCRIPTION
**Pull request by kollil**
#### Rationale
Added Prima DB schema.

### CODE REVIEW

**Review by Ohsudev — APPROVED** [review]

File: onprc_ehr/resources/schemas/onprc_ehr.xml [hunk 1]
```diff
@@ -998,33 +998,250 @@
...
+
+    <table tableName="Prima_CassetteEventLocations" tableDbType="TABLE">
+        <columns>
+            <column columnName="CassetteEventId" />
+            <column columnName="LabStationTypeId" />
+            <column columnName="LocationId" />
+            <column columnName="WorkstationId" />
+            <column columnName="Created" />
+            <column columnName="PersonId" />
+    </table>
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/schemas/onprc_ehr.xml:1202]: This is missing the closing </column>, and is causing a variety of test failures. Your dev machine should be writing the same errors to the console in the debugger or in labkey.log.

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]

**Comment by kollil** [comment]
@labkey-jeckels , Please review & approve my changes so I can create a default branch.

Thanks
Lakshmi

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
Thanks for adding the missing </columns>

The test results are showing that there are discrepancies between the tables in the database and the XML metadata.

You can see this and get details by digging into the failure message for SchemaXMLTestCase in TeamCity

You can also reproduce this failure locally, which may be faster to troubleshoot and correct.
Go to http://localhost:8080/labkey/junit-begin.view, find SchemaXMLTestCase, and click on it to run that test. You'll see the errors. You can make corrections to onprc_ehr.xml and rerun the test by reloading the page in the browser.

**Comment by kollil** [comment]
@labkey-jeckels,
Thank you Josh. The SchemaXMLTestCase link helped fixing the xml issues. I fixed the Prima related data issues, because it's the new development. But, There are others also need fixes. All the others are already in production. 
They are,
1. The following are eIACUC data issues, generated by Gary's etl process. The process is already in production,
ERROR: TableName "project" type "TABLE" found in XML but not in database.
ERROR: TableName "PRIME_VIEW_ANIMAL_GROUPS" type "TABLE" found in XML but not in database.
ERROR: TableName "protocol" type "TABLE" found in XML but not in database.
ERROR: TableName "PRIME_VIEW_PROTOCOL_Processing" type "TABLE" found in XML but not in database.
ERROR: TableName "PRIME_VIEW_NON_SURGICAL_PROCS" type "TABLE" found in XML but not in database.
ERROR: TableName "PRIME_VIEW_PROTOCOLS" type "TABLE" found in XML but not in database.
ERROR: TableName "PRIME_VIEW_SURGICAL_PROCS" type "TABLE" found in XML but not in database.

2. Housing transfer alert module, it's also in production,
ERROR: Table "availablecages_temp" missing from XML.
ERROR: Table "availablecagesbyroom_temp" missing from XML.
ERROR: Table "roomutilization_temp" missing from XML.

I can fix the "2" issues, because it was deployed by me into production few mons ago.
I am not sure if I should be fixing the ones already in production. Please advice.

Thanks
Lakshmi

**Comment by labkey-jeckels** [comment]
@kollil your feature branch is missing the fixes that have already been made in release20.11-SNAPSHOT because it was branched from the state of the code back in May. Notice the big gap in dates when you look at the history here:

https://github.com/LabKey/onprcEHRModules/commits/20.11_fb_PrimaReports

To minimize the odds of this in the future, be sure that you've pulled the latest changes from the release branch before you create your feature branch.

To address this in your current feature branch, you can merge in the changes from the release20.11-SNAPSHOT branch as I showed during our July 6 call. I checked the recording and it's about 45 minutes in.

https://www.labkey.org/ONPRC/wiki-page.view?name=trainingSessionRecordings

**Comment by kollil** [comment]
Hmmm, Interesting, I always make sure my project is upto date before I create my fb. I will check again submit the changes. Thanks for the prompt response.

**Comment by kollil** [comment]
@labkey-jeckels , 
When I tried to merge the latest onprc_ehr.xml file from 20.11_snapshot it included many other files.
I fixed the xml file and pushed the changes, it included all the other files too. Could you please exclude them before creating the default branch.
Thank you!

**Review by labkey-jeckels — APPROVED** [review]
@kollil Don't worry about those extra files. They're just catching your feature branch up with the release20.11-SNAPSHOT. If you look at the Files tab here in the pull request, you'll see that it's only showing the 6 files you actually edited or added. The tests are now happy. Nice job on the merge!

**Comment by kollil** [comment]
Thanks Josh.

**Pull request merged**